tests/test_minimizers.py

```diff
@@ -27,6 +27,35 @@ def chi_squared(x, y, a, b, sum=True):
 def worker(fit_obj):
     return fit_obj.execute()
 
+class SqrtLeastSquares(LeastSquares):
+    """
+    Minimizes the square root of LeastSquares. This seems to help SLSQP in
+    particular, and is considered good practise since the square can grow to
+    rapidly, leading to numerical errors.
+    """
+    # TODO: Make this a standard objective, and perhaps even THE standard
+    # objective. This lightweight version is given without proper testing
+    # because only the call is relevant, and this makes our multiprocessing test
+    # work.
+    def __call__(self, *args, **kwargs):
+        chi2 = super(SqrtLeastSquares, self).__call__(*args, **kwargs)
+        return np.sqrt(chi2)
+
+    def eval_jacobian(self, *args, **kwargs):
+        sqrt_chi2 = self(*args, **kwargs)
+        chi2_jac = super(SqrtLeastSquares, self).eval_jacobian(*args, **kwargs)
+        return 0.5 * (1 / sqrt_chi2) * chi2_jac
+
+    def eval_hessian(self, *args, **kwargs):
+        sqrt_chi2 = self(*args, **kwargs)
+        sqrt_chi2_jac = self.eval_jacobian(*args, **kwargs)
+        chi2 = super(SqrtLeastSquares, self).__call__(*args, **kwargs)
+        chi2_jac = super(SqrtLeastSquares, self).eval_jacobian(*args, **kwargs)
+        chi2_hess = super(SqrtLeastSquares, self).eval_hessian(*args, **kwargs)
+        # Hessian Approximation! Just takes the
```

Comment:
missing some words?

Reply:
I removed it all together, it was outdated since I actually wrote down the full hessian.